docs/content/components/tooltip.md

```diff
@@ -27,7 +27,7 @@ import Example from '../../src/@primer/gatsby-theme-doctocat/components/example'
 
 ### Default
 
-<Example src="<div class='pt-5'>  <span aria-label='Even bolder' data-view-component='true' class='tooltipped tooltipped-n'>Default Bold Text</span></div>" />
+<Example src="<div class='pt-5'>  <span data-view-component='true' class='tooltipped tooltipped-n'>Default Bold Text</span></div>" />
```

Comment:
Given `aria-label` is used for the current tooltip implementation, I think we need to keep it here and update examples to use `button` or some other supported element.

Reply:
hmm I wonder if we should add an exception for Tooltip until we figure out the new version, wdyt?

Reply:
yeah, I think we should just deprecate `Tooltip`. 😅

Reply:
I'm gonna skip the check for Tooltip, when you propose your new version here we can deprecate the old one :)